code/modules/projectiles/guns/projectile.dm

```diff
@@ -124,10 +124,7 @@
 
 /obj/item/weapon/gun/projectile/process_chambered()
 	var/obj/item/ammo_casing/AC = getAC()
-	if(in_chamber)
-		return 1 //{R}
-	if(isnull(AC) || !istype(AC))
-		return
+	can_discharge()
```

Comment:
So... `can_discharge()` in that block won't cause `process_chambered()` to return whatever it's value is, instead it just makes `can_discharge()` return a number that doesn't do anything?